bin/apiml_cm.sh

```diff
@@ -336,11 +338,65 @@ public class ExportPrivateKey {
     }
 }
 EOF
+    else
+        cat <<EOF >$TEMP_DIR/ExportPrivateKey.java
```

Comment:
you should add a code comment so that this code is maintained over time:
why is there a nearly duplicate java function? because the import needed for z/os keyring support will fail in docker or in development environments. this section is for docker and does not support keyrings.

Reply:
Rather than the copy-paste of code, I wonder if it would be better to have a z/OS specific subclass of private key given that the differences seem to be that the z/os version has a little bit of extra logic?

Reply:
The problem I saw was that you can't do a conditional import statement, and I think the zos-only import was an issue leading to a crash. We can improve upon this if that's not the case.